backend/api/views/news.py

```diff
@@ -23,7 +25,8 @@ class NewsViewSet(viewsets.ViewSet):
     def list(self, request):
         """List news"""
         serializer = self.serializer_class(self.queryset, many=True)
-        return Response(serializer.data)
+        response = sorted(serializer.data, key=lambda item: datetime.strptime(item["date"], "%Y-%m-%d"), reverse=True)
```

Comment:
Sorting should be done at queryset level : this way we can leverage all the power the database has for these kinds of operations.